src/components/common/StaffPayment.vue

```diff
@@ -49,17 +49,13 @@ export default class StaffPayment extends Vue {
   /** Prop to provide section number. */
   @Prop({ default: '' }) readonly sectionNumber!: string
 
-  /** Whether staff payment is invalid, only when prompted by app. */
+  /** Whether sub-component should show invalid section styling. */
   get invalidStaffPayment (): boolean {
-    return this.getAppValidate && !this.getFlagsReviewCertify.isValidStaffPayment
+    return (this.getAppValidate && !this.getFlagsReviewCertify.isValidStaffPayment)
   }
 
-  /** True when prompted by the app AND the user has selected an option. */
-  get validateStaffPayment (): boolean {
-    return this.getAppValidate && !!this.getStaffPayment?.option
-  }
```

Comment:
This is no longer needed. The shared Staff Payment component should validate itself when the app-validate flag is set.

This was probably left over from the initial implementation of the shared Staff Payment component, but since then I re-worked its internal validation to work better (and simpler).